langchain/document_loaders/fauna.py

```diff
@@ -0,0 +1,56 @@
+from typing import List, Optional
+from langchain.docstore.document import Document
+
+from langchain.document_loaders.base import BaseLoader
+
+
+class FaunaLoader(BaseLoader):
+    def __init__(
+        self,
+        query: str,
+        page_content_field: str,
+        secrect: str,
+        metadata_fields: Optional[List[str]] = None,
```

Comment:
Favor Sequence over List for inputs to promise immutability of collection

```suggestion
        metadata_fields: Optional[Sequence[str]] = None,
```